lwr_controllers/include/lwr_controllers/gravity_compensation.h

```diff
@@ -0,0 +1,41 @@
+#ifndef GRAVITY_COMPENSATION_H
+#define GRAVITY_COMPENSATION_H
+
+#include "KinematicChainControllerBase.h"
+
+#include <std_msgs/Bool.h>
+#include <std_msgs/Float32.h>
+
+namespace lwr_controllers
+{
+    class GravityCompensation: public controller_interface::KinematicChainControllerBase<hardware_interface::PositionJointInterface>
```

Comment:
That's why controllers typically list the joint names in YAML files, and then retrieve them from the HW interface, see for instance how the ubiquitous forward command controller does it [here](https://github.com/ros-controls/ros_controllers/blob/indigo-devel/forward_command_controller/include/forward_command_controller/forward_command_controller.h#L77-L83)